lib/dor_indexing/indexers/descriptive_metadata_indexer.rb

```diff
@@ -29,7 +29,8 @@ def to_solr
 
           # contributor
           'author_text_nostem_im' => author_primary, # primary author tokenized but not stemmed
-          'sw_author_tesim' => author_primary, # used for author display in Argo
+          'sw_author_tesim' => author_primary, # DEPRECATED - used for author display in Argo
```

Comment:
Minor request: `# DEPRECATED - previously used for author display in Argo`

Reply:
it's still used and will be until the new field is populated.   Which now takes 5 days or so.